### ISSUE TITLE
Consider making "File settings" options global

### ISSUE THREAD

**Issue by elyahw** [gsantner/markor#2212]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **enhancement/feature request**. Not a [bug or question](https://github.com/gsantner/markor/issues/new/choose).
- [X] The topic is **not** already reported at [Issues](https://github.com/gsantner/markor/issues?q=#js-issues-search). _(I've searched it)_.
- [X] Markor **is** up to date. See [Releases](https://github.com/gsantner/markor/tags) for the latest version. Updates are available from [F-Droid](https://f-droid.org/en/packages/net.gsantner.markor/) and GitHub.
- [X] The wanted feature/enhancement is not present in the latest development version (git master). (Please [download](https://nightly.link/gsantner/markor/workflows/build-android-project/master) and try the test version of Markor, named **Marder**. Don't worry; Markor and Marder appear as completely separate applications. You can install both side-by-side, and Markor's settings are not touched. If your desired feature is present, you don't need to open this issue. The change will be part of the next Markor update.)

### Description

1. Currently if you open a note, press '...' on the top right corner, then "File settings", there are many useful options such as: "Wrap words", "Line numbers", "Syntax highlighting", "Font size".
2. They are very useful options. Some of them are disabled by default such as "Line numbers", which is particularly useful to me, and which I have to enable it manually on many individual files.
3. Why not move these options to the global settings so that they are applied to a certain extension?

**Advantages:**
1. It will declutter the main interface, and move 4 buttons to the settings. Therefore simplifying the main interface. Perhaps consider then reshuffling the '...' menu?
2. All four options I quoted are very useful that it should be possible to set them globally, not individually.

### Information

Android version: 13
Device: Pixel 5a
App Version: 2.11.1

### Source

GitHub Releases

### Format / File type

Not specific

### Additional info / Log

```shell
-
```

**Comment by gsantner** [comment]
You can find the default setting in the normal settings menu, i.e. under Editor section. That should be the case for the majority.

**Comment by guanglinn** [comment]
@elyahw , I basically agree with your opinions. Such as Line numbers, I also think it should have a global setting, and this option initially did support global settings, you can see this [PR](https://github.com/gsantner/markor/pull/2062#discussion_r1218628884/).

**Comment by elyahw** [comment]
Thanks. @gsantner But there are no global options for "Line numbers" and "Syntax highlighting".

I suggest to add these two options to the rest of global settings, and perhaps remove the file-specific ones.

My purpose to simplify the app. Right now I think it has too many options that can be accessed from different places, therefore it is confusing (at least to me, a new user to the app).

**Comment by harshad1** [comment]
The way we do this for other cases is have a global default for new files and have the per-file setting which overrides this if toggled.

I think such a global default option for syntax highlighting and line numbers is a good idea

**Comment by elyahw** [comment]
Yes. I have been thinking about it. I think the per-file settings should be removed. Why?

1. Why not use a different extension and enable line numbers on that extension only. This is a simple solution.
2. It over complicates things: There are many extensions and each comes with a default set of settings. I have 100s of notes on my phone I keep deleting/cutting/pasting them into each other. There is no way I can track the settings for an individual note. Using the global settings for this purpose is a peace of mind.

**Comment by gsantner** [comment]
I would also like to simplify it a bit, from my view - only make two pairs of global settings for the majority. Which is for files in normal editor/viewer, and for the other part how it appears on the main screen. You want less bloat there, i.e. no line numbers or alike.

What can't be global and at the end is (or should be) really file specific are things like BOM and file encryption.

**Comment by HaleyHalcyon** [comment]
https://github.com/gsantner/markor/discussions/2112

I want line numbers to be visible by default at the time of opening a new Markdown file, so that I know where the line breaks are. Presently, Markor has no option for that.
